### PR TITLE
update boto requirements to address issue #87

### LINKED ISSUE #87: ClientError: An error occurred (403) when calling the HeadBucket operation: Forbidden
Encountered `ClientError: An error occurred (403) when calling the HeadBucket operation: Forbidden
` when trying `cc.get_interface()`. The issue was solved by changing the versions of my botocore and boto3. 
I downgraded botocore from 1.29.63 to 1.19 and boto3 from 1.26.63 to 1.15.

### PR DESCRIPTION
**Pull request by jenholmberg**
included inequality 

Fixes #87

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,7 +1,8 @@
 wheel>=0.31.1
 numpy>=1.6.0
 scipy>=0.9.0
-boto3>=1.2.3
+botocore<=1.19
+boto3<=1.15
```
Comment (TomDLT) [inline requirements.txt:5]: It is probably better to keep a minimum version too.
You can have both inequalities with the syntax `boto3>=1.2.3, <=1.15`

**Comment by TomDLT** [comment]
Btw, you can automatically close the associated issue by writing `Fixes #87` in the top issue message. It also creates a handy hyperlink to the issue.

File: requirements.txt (hunk 1 above)
Comment (jenholmberg) [reply requirements.txt:5]: Ah, that makes sense. I'll keep the boto3 minimum version too. There wasn't a botocore inequality before, so I'll just leave that as is.

File: requirements.txt [hunk 2]
```diff
@@ -1,7 +1,8 @@
 wheel>=0.31.1
 numpy>=1.6.0
 scipy>=0.9.0
-boto3>=1.2.3
+botocore<=1.19
+boto3>=1.15, <=1.2.3
```
Comment (TomDLT) [inline requirements.txt:5]:
```suggestion
boto3>=1.2.3, <=1.15
```

**Review by micwinter — APPROVED** [review]
Looks good for merging

**Pull request merged**